perl/PKGBUILD

```diff
@@ -1,15 +1,13 @@
 # Maintainer: Alexey Pavlov <Alexpux@gmail.com>
 
-pkgname=perl
+pkgname=('perl' 'perl-devel')
```

Comment:
Add `pkgbase=perl`

Reply:
Must have missed that on the python changes, tried to replicate them.